src/environment.cpp

```diff
@@ -425,6 +426,15 @@ bool ServerEnvironment::line_of_sight(v3f pos1, v3f pos2, float stepsize, v3s16
 	return true;
 }
 
+void ServerEnvironment::kickAllPlayers()
+{
+	for (std::vector<Player*>::iterator it = m_players.begin();
+			it != m_players.end();
+			++it) {
+		((Server*)m_gamedef)->DenyAccess_Legacy((*it)->peer_id, utf8_to_wide("Server shutting down..."));
```

Comment:
in that case instead of renaming the function, make it so the `Server shutting down...` message is configurable so that it can be reused for other purposes in the future ??